v-next/hardhat/src/internal/cli/telemetry/sentry/anonymizer.ts

```diff
@@ -254,6 +287,8 @@ export class Anonymizer {
       result.stacktrace = await this.#anonymizeStacktrace(value.stacktrace);
     }
 
+    result.mechanism = value.mechanism;
```

Comment:
Could this have been done in `results` literal initializer?

Reply:
Done